setup.py

```diff
@@ -42,6 +46,9 @@ def read(*filenames, **kwargs):
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```

Comment:
If we include exact minors, won't we have the same problem as #13 where someone might want `2.6` or `3.3` support and assume we don't have it?